src/Lykke.Job.BlockchainCashinDetector.Services/StartupManager.cs

```diff
@@ -13,8 +16,26 @@ namespace Lykke.Job.BlockchainCashinDetector.Services
     [UsedImplicitly]
     public class StartupManager : IStartupManager
     {
+        private readonly ILog _log;
+        private readonly IEnumerable<IDepositWalletsBalanceProcessingPeriodicalHandler> _depositWalletsBalanceProcessingHandlers;
+
+        public StartupManager(
+            ILog log, 
+            IEnumerable<IDepositWalletsBalanceProcessingPeriodicalHandler> depositWalletsBalanceProcessingHandlers)
+        {
+            _log = log.CreateComponentScope(nameof(StartupManager));
```

Comment:
Here and below in similar situations: I'd recommend to introduce some check for null-valued input params standing for DI instances.